core/src/main/scala/org/ensime/core/AnalyzerManager.scala

```diff
@@ -183,7 +133,23 @@ class AnalyzerManager(
 
     }
 
-  private def withExistingModuleFor(file: Path, req: RpcAnalyserRequest)(f: (RpcAnalyserRequest, EnsimeProject) => Unit): Unit = {
+  def toFile(f: SourceFileInfo) = f.file match {
+    case RawFile(path) => path.toFile
+    case ArchiveFile(path, _) => path.toFile
```

Comment:
Well because `RemoveFileReq` accepts a parameter of type `File` and here I have a list of `SourceFileInfo` . I'm not sure what I should do?